resbaz/resbazTucson2025.qmd

```diff
@@ -0,0 +1,111 @@
+---
+layout: default
+title: ResBaz Arizona
+css: /styles/festivalPages.css
+aliases:
+  - resbazArizona2025
+---
+
+<script type="text/javascript" src="https://latest.cactus.chat/cactus.js"></script>
+<link rel="stylesheet" href="https://latest.cactus.chat/style.css" type="text/css">
+
+
+The Research Bazaar is a worldwide festival promoting the digital literacy emerging at the center of modern research. Throughout 2025, events are happening at university campuses around the globe.
+
+ResBaz Arizona is a free intensive conference happening **Tursday, May 29th 2024** where researchers come together to up-skill in "next generation digital research tools and skills." In the spirit of a marketplace or bazaar, ResBaz is a highly participatory event where researchers from many different disciplines can learn, share knowledge and skills, and have fun!
+
+Who should attend? Researchers at ALL levels from ALL disciplines. PhD and research master's students are especially encouraged to attend. Research technologists who support researchers to do their job should also come along!

```

Comment:
```suggestion
Who should attend? Researchers at ALL levels, from ALL disciplines, from ALL institutions and organizations, as well as solo researchers. PhD and research master's students are especially encouraged to attend. Research technologists who support researchers to do their job should also come along!

```

Reply:
Mostly because I read this paragraph and didn't see myself in it. But I know I'm a weirdo on the fringes of research, so rewording / just skipping this change is just fine